website/docs/cli/commands/init.mdx

```diff
@@ -85,7 +85,7 @@ The `-migrate-state` option will attempt to copy existing state to the new
 backend, and depending on what changed, may result in interactive prompts to
 confirm migration of workspace states.  The `-force-copy` option suppresses
 these prompts and answers "yes" to the migration questions. This implies
-`-migrate-state`.
+initiating the new backend with the existing state.
```

Comment:
I think there's a disconnect here between the intent of the original documentation and your proposal. What I'd suggest instead here to clarify the text which caused the misunderstanding is replacing the "This implies […]" sentence with something like:

> Enabling `-force-copy` also automatically enables the `-migrate-state` option.